crates/rspack_binding_options/src/js_values/union.rs

```diff
@@ -0,0 +1,59 @@
+use std::fmt::Debug;
+
+use napi::{NapiRaw, NapiValue};
+
+/// The concept is similar to `Either`, but `Either` is a common used name. To avoid conflictcations I
```

Comment:
I don't if this is necessary, as this would cause type generation problem.

Reply:
`napi::Either` could not be used as a field of `#napi[object]`. So this is necessary.